Win10_VirtualDesktop_Optimize.ps1

```diff
@@ -82,114 +103,205 @@ The Store and a few others, such as Wallet, were left off intentionally.  Though
 it is nearly impossible to get it back.  Please review the lists below and comment out or remove references to packages that you do not want to remove.
 #>
 
+$VerbosePreference = "Continue"
+$WindowsVersion = (Get-ItemProperty "HKLM:\Software\Microsoft\Windows NT\CurrentVersion\").ReleaseId
```

Comment:
We need to be able to deviate from the specific version of windows.  This is pointing them to the configuration folder, but does not allow for a choice of a custom folder where they might have made their own Config files

Reply:
Maybe we update the param block and allow the user to select a windows version of their choice, but the default value would be the version pulled back from the registry.  Thoughts?